joomla/joomla-cms
# [6.0] Smart Search: Removing dependency on CMSObject

### Summary of Changes
The `CMSObject` class has been deprecated for a long time and for 6.0, we want to remove it as much as possible. This PR removes it from Smart Search.


### Testing Instructions
Apply the PR and run the indexer, look at statistics and search in the frontend and see, that everything still works as expected.


### Link to documentations
Please select:

- [X] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/380
